src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -7,5 +7,6 @@ public enum DocumentFlags
     {
         None = 0,
         Versioned = 1,
+        Artificial = 2,
```

Comment:
You're right about your concern, but it looks like artificial would be better to describe a document which isn't going to be versioned or replicated. Note that the flags name should be exposed to the user as read only in the metadata.